src/OrbitGl/App.cpp

```diff
@@ -345,32 +345,36 @@ void OrbitApp::OnTracepointEvent(orbit_client_protos::TracepointEventInfo tracep
       is_same_pid_as_target);
 }
 
-void OrbitApp::UpdateModulesAbortIfModuleWithoutBuildIdReloader(
+void OrbitApp::UpdateModulesAbortCaptureIfModuleWithoutBuildIdNeedsReload(
     absl::Span<const ModuleInfo> module_infos) {
-  auto updated_modules = module_manager_->AddOrUpdateModules(module_infos);
+  auto updated_modules = module_manager_->AddOrUpdateNotLoadedModules(module_infos);
 
   if (!updated_modules.empty()) {
     std::string module_paths;
     for (const auto* updated_module : updated_modules) {
       if (!module_paths.empty()) module_paths += ", ";
       module_paths += updated_module->file_path();
```

Comment:
This for loop can now be a `absl::StrJoin(", ")` (or however the abseil function is called)

Reply:
StrJoin works on snap of strings which is not the case here. we are operating on vector of ModuleDatas